homework7/hw1.py

```diff
@@ -0,0 +1,54 @@
+"""
+Given a dictionary (tree), that can contains multiple nested structures.
+Write a function, that takes element and finds the number of occurrences
+of this element in the tree.
+
+Tree can only contains basic structures like:
+    str, list, tuple, dict, set, int, bool
+"""
+from typing import Any
+
+# Example tree:
+example_tree = {
+    "first": ["RED", "BLUE"],
+    "second": {
+        "simple_key": ["simple", "list", "of", "RED", "valued"],
+    },
+    "third": {
+        "abc": "BLUE",
+        "jhl": "RED",
+        "complex_key": {
+            "key1": "value1",
+            "key2": "RED",
+            "key3": ["a", "lot", "of", "values", {"nested_key": "RED"}],
+        },
+    },
+    "fourth": "RED",
+    "set_some": ("RED", "1", None, 6, True, (1, "RED")),
+}
+
+
+def find_occurrences(tree: dict, element: Any) -> int:
+    """
+    Find amount of element in dict(tree)
+    param:
```

Comment:
there is no need in params in docstrings if you have verbose representative names and typings